app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -1398,8 +1398,11 @@ class BrowserTabFragment : Fragment(), FindListener, CoroutineScope, DaxDialogLi
             menuButton?.isVisible = viewState.showMenuButton
 
             if (viewState.fireButton.playPulseAnimation()) {
+                appBarLayout.setExpanded(true, true)
```

Comment:
@marcosholgado could you review this commit?

Related to: https://app.asana.com/0/1142021229838617/1198200366504622/f

_"Could we pin the URL bar just in this scenario? The same logic that clears the blue circle animation would also clear the pin."_